tools/index-updater.awk

```diff
@@ -133,7 +155,7 @@ END {
 		    level = level - 1;
 		}
 		print "";
-		print "  <h3>" letter_new "</h3>";
+		print "  <h3> Letter " letter_new "</h3>";
```

Comment:
I don't find that "Letter" adds to the readability of the index; I rather prefer
`A`
to
`Letter A`

Reply:
The reason for "Letter " is covered in the PR description, but I also don't love what I did. I have an idea how to solve the underlying issue, and will do so in a follow change.